packages/@uppy/dashboard/src/components/Dashboard.js

```diff
@@ -127,7 +127,7 @@ module.exports = function Dashboard (props) {
               <strong className="uppy-Dashboard-serviceMsg-title">
                 {props.i18n('sessionRestored')}
               </strong>
-              <div class="uppy-Dashboard-serviceMsg-text">
+              <div className="uppy-Dashboard-serviceMsg-text">
```

Comment:
Is this a linter thing? If so, should we keep this rule? What's the upside of using `className` over `class`?

Reply:
You have to use `className` in React/Preact.